Applications/TransferLearning/How-to-gen-tflite-model.md

```diff
@@ -0,0 +1,56 @@
+# How to generate TFlite model for feature extractor
+
+In order to use tflite model as an feature extractor, we need to remove some of the layers from pre-trained model and convert to tflite model.
```

Comment:
as `a` feature extractor, we `might` need to remove some of the `last layers` from the `Tensorflow's` pre-trained model `while` converting to tflite model `format`.

Reply:
Actually there are some more grammatical changes required. I will do in another PR, lets go forward with this.